src/APP/office/officesPage.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import './officesPage.scss'
+import OfficeListItem from './officeListItem';
+import OfficeGridItem from './officeGridItem';
+import MapPage from './MapPage';
+
+const OfficesPage = (props) =>{
+    
+    const officesDivClass = (props.view === 'list' || props.view === 'grid') ? ((props.view === 'list') ?'officesDiv' : 'officesDivGrid') : 'mapDiv';

```

Comment:
Too many magic strings use constant for it. 
This logic screams for react router.